backend/packages/Upgrade/src/api/DTO/ExperimentDTO.ts

```diff
@@ -463,6 +464,9 @@ export class ExperimentDTO {
   @IsNotEmpty()
   @IsEnum(EXPERIMENT_TYPE)
   public type: EXPERIMENT_TYPE;
+
```

Comment:
I don't know how to add validation for this... this interface will have a property called "moocletDetails.policyParameters" that will contain what is basically a config object that is different for each algorithm that we will support. See "policyParameters" interfaces in the Mooclet types file for examples.

Reply:
i will try using discriminator in follow-up PR